### DISCUSSION TITLE
Light and dark theme splash screen

### DISCUSSION THREAD

**Discussion by tommag21** [NativeScript/NativeScript#10152] [Q&A]
Has anyone implemented a different splash screen for light and dark theme on both iOS and Android? I couldn't find any discussion regarding a NativeScript implementation, only platform-specific examples.

**Comment by vallemar** [comment]
@tommag21 You need to do it the same as it is done on the platform (android or ios) if you are looking for how to do it on android for example it should be the same on NS
